exercises/pangram/src/test/java/PangramsTest.java

```diff
@@ -66,4 +66,10 @@ public void pangramWithNonAsciiCharacters() {
     public void panagramInAlphabetOtherThanAscii() {
         assertFalse(Pangrams.isPangram("Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства."));
     }
+
+    @Ignore
+    @Test
+    public void upperAndLowerCaseVersionsOfTheSameCharacterShouldNotBeCountedSeparatelyIsNotPangram() {
```

Comment:
Ok I can see what you meen but I can't understand what we actually test in this case. The test is supposed to check if same chars are counted seperately but I don't see how we check that in this test... For example we have "pangram with mixed case and punctuation" test case that checks if a pangram with upper and lower case chars is considered true by counting upper and lowercase letters but I think that we don't count anything here. So my question is "Are we actually test anything with this test?"